docs/releases/1.9-NOTES.md

```diff
@@ -309,7 +309,7 @@ None known at this time
 * Typo fix "etcd manager"->"etcd-manager" [@AdamDang](https://github.com/AdamDang) [#4628](https://github.com/kubernetes/kops/pull/4628)
 * Typo fix "provides"->"provide" [@AdamDang](https://github.com/AdamDang) [#4622](https://github.com/kubernetes/kops/pull/4622)
 * Typo instance group ->"InstanceGroup" [@AdamDang](https://github.com/AdamDang) [#4621](https://github.com/kubernetes/kops/pull/4621)
-* Typo "offical"->"official" [@AdamDang](https://github.com/AdamDang) [#4637](https://github.com/kubernetes/kops/pull/4637)
+* Typo "official"->"official" [@AdamDang](https://github.com/AdamDang) [#4637](https://github.com/kubernetes/kops/pull/4637)
```

Comment:
Well this is a bit of a problem... I think we can probably live with it here, but is there any way to skip certain lines?

Reply:
yeah .. I'd like to keep the releases directory in there .. call this a one off :-)